bytebase/bytebase.com
# refactor: buglist fixes

This pull request brings fixes from buglist 2 and medium priority fixes.

**Steps to check:**
- open website ([home](https://bytebase-git-buglist-2-bytebase-pixelpoint.vercel.app/), [about](https://bytebase-git-buglist-2-bytebase-pixelpoint.vercel.app/about/), [docs regex](https://bytebase-git-buglist-2-bytebase-pixelpoint.vercel.app/docs/vcs-integration/github-com/), [docs youtube](https://bytebase-git-buglist-2-bytebase-pixelpoint.vercel.app/docs/get-started/quick-start/))
- make sure that everything looks and feels as expected
